build/test-replacements.js

```diff
@@ -138,7 +142,26 @@ module.exports['common.js'] = [
     [
       /var regexp = `\^\(\\\\w\+\)\\\\s\+\\\\s\$\{port\}\/\$\{protocol\}\\\\s`;/,
       `var regexp = '^(\\w+)\\s+\\s' + port + '/' + protocol + '\\s';`
-    ]
+    ],
+    [
+        /require\(['"]stream['"]\)/g
+      , 'require(\'../\')'
+    ],
+    [/forEach\(data, line => \{\n\s+this\.emit\('data', line \+ '\\n'\);\n\s+\}\);/m,
```

Comment:
there is an arrow function here that I needed to rewrite, if we get too many of them we may want to consider using a transpiler for future builds